src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1110,10 +1110,48 @@ private async Task SendInviteAsync(OrganizationUser orgUser, Organization organi
                 throw new BadRequestException("Invalid token.");
             }
 
+            ICollection<Policy> orgPolicies = null;
+            ICollection<Policy> userPolicies = null;
+            async Task<bool> hasPolicyAsync(PolicyType policyType, bool useUserPolicies = false)
+            {
+                ICollection<Policy> policies = null;
+                if (useUserPolicies) 
+                {
+                    if (userPolicies == null)
+                    {
+                        userPolicies = await _policyRepository.GetManyByUserIdAsync(user.Id);
+                    }
+                    policies = userPolicies;
```

Comment:
Not that you _should_, but could make this a 1-liner 😁 :
```c#
policies = userPolicies = userPolicies ?? await _policyRepository.GetManyByUserIdAsync(user.Id);
```

Reply:
Done + reduced line count even more by using a ternary